kong/db/migrations/core/init.lua

```diff
@@ -2,4 +2,5 @@ return {
   "000_base",
   "001_14_to_15",
   "002_15_to_1",
+  "003_1.0.0_to_1.1.0",
```

Comment:
What convention should we follow?
1. `003_1.0.0_to_1.1.0`
2. `003_1_to_1_1`
3. `003_1_to_1.1`
4. `003_1_to_11`

I think when I did a PR that added first `002_15_to_1`, I named it as `002_15_to_10`, but someone renamed it to `002_15_to_1`. My reasoning was to always use two digits, and never semver beyond 9 (and just leave `001` and `002` as exception to this rule). Did someone have an idea to this?

Reply:
5. `003_100_to_110` is the most precise, concise, and future-proof notation (e.g. introducing migrations in future patch releases as elaborated in my previous comment).

> and never semver beyond 9

No big deal to do so. Besides, if mixing up digits is a concern, we should consider using an integral value in these file names (like [kong.version_num](https://github.com/Kong/kong/blob/master/kong/global.lua#L12-L15)).

I used `1` and not `10` in the 1.0 release so as to avoid confusion with `0.10`, but if we use `110`, and if we agree to this convention, this is not really a concern anymore.

Reply:
Let's do `110` then. This is just for human visual inspection. Nobody's going to get `4312` confused between `4.31.2` or `4.3.12` given that the files are ordered in sequence with prefixes like `004_` anyway.

Reply:
Ok, doing `110`.

When we get to Kong v10 migrations will probably have changed again anyway :)